src/Entities/DeliveryAddress.php

```diff
@@ -12,8 +14,29 @@ class DeliveryAddress extends BaseEntity
 
     protected string $resource = DeliveryAddressResource::class;
 
-    public function get(string $identifier): ?DeliveryAddressResource
+    /**
+     * @return Collection<DeliveryAddressResource>|null
+     */
+    public function get(string $identifier)//: ?Collection
```

Comment:
```suggestion
    public function get(string $identifier): ?Collection
```

Reply:
Comment has been removed. Added it while I was testing and completely forgot to remove it afterwards.